libs/zzz/page-optimize/src/BuildsDisplay.tsx

```diff
@@ -53,16 +54,18 @@ function Build({
   baseStats: Stats
 }) {
   const { database } = useDatabaseContext()
+  const character = useCharacterContext()
   const sum = useMemo(
     () =>
       applyCalc(
         baseStats,
+        character?.conditionals ?? {},
```

Comment:
semi-unrelated to PR:
changing conditionals causes the displayed stat to change on generated builds, but does not change the calculated value.

should we consider just grabbing the final result from the calculation instead of cache?